src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const partnerLabel = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+      {age && <p className="Person__age">I am {age}</p>}
+      {isMarried ? (
+        <p className="Person__partner">
+          {partnerName} is my {partnerLabel}
+        </p>
+      ) : (
+        <p className="Person__partner">I am not married</p>
+      )}
```

Comment:
```suggestion
  const partnerLabel = sex === 'm' ? 'wife' : 'husband';
  const personPartner = isMarried ? `${partnerName} is my ${partnerLabel}` : 'I am not married'

  return (
    <section className="Person">
      <h2 className="Person__name">My name is {name}</h2>
      {age && <p className="Person__age">I am {age}</p>}
        <p className="Person__partner">
          {personPartner}
        </p>
```

it is redundant to rerender `<p className="Person__partner">`. enough to rerender text inside.